src/components/InputandFilterPanel/InputandFilterPanel.styles.tsx

```diff
@@ -67,16 +67,57 @@ export const InputImage = styled.img`
 	max-height: 15px;
 `;
 
-export const StyledCountriesSelectLabel = styled.label`
+export const StyledCountriesLabel = styled.label`
 	border-radius: 5px;
 	width: 20%;
 	min-width: 145px;
 	background-color: ${(props) => props.theme.colors.white};
 	text-align: center;
 `;
 
-export const StyledCountriesSelect = styled.select`
+export const StyledFilterPanel = styled.div`
+	border-radius: 5px;
+	background-color: ${(props) => props.theme.colors.white};
+	width: 15%;
+	min-width: 170px;
+	display: flex;
+	justify-content: space-around;
+	padding: 20px 10px;
+	box-shadow: 2px 2px 10px -8px rgba(66, 68, 90, 1);
+	font-family: ${(props) => props.theme.fonts.family.basic};
+	font-weight: ${(props) => props.theme.fonts.weight.semiBold};
+	font-size: ${(props) => props.theme.fonts.sizes.s};
+	position: relative;
+`;
+
+export const FilterButton = styled.button`
 	border: none;
-	width: 90%;
-	padding: 18px 5px;
+	outline: none;
+	background-color: ${(props) => props.theme.colors.white};
+`;
+export const StyledImg = styled.img`
+	max-width: 12px;
+	max-height: 12px;
+`;
+
+export const StyledDivWithOption = styled.div`
+	margin-top: 42px;
+	display: block;
+	position: absolute;
+	background-color: ${(props) => props.theme.colors.white};
+	min-width: 160px;
+	box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
+	z-index: 1;
+	width: 100%;
+	border-radius: 5px;
+	padding: 10px 0;
+`;
+
+export const CountryOption = styled.p`
```

Comment:
hint - You render a LIST of items not a container of paragraphs ;)

Reply:
fixed.